frontend/website/src/components/home/category-card.svelte

```diff
@@ -1,40 +1,46 @@
 <script lang="ts">
 	import { viewersToString } from "$/lib/utils";
-	import ViewersGuy from "./icons/viewers-guy.svelte";
+	import Fa from "svelte-fa";
+	import { faUser } from "@fortawesome/free-regular-svg-icons";
 
 	export let title: string;
 	export let image: string;
 	export let viewers: number;
 </script>
 
-<a class="category" href={`/categories/${title.toLowerCase()}`}>
-	<img src={image} alt={title} />
+<a
+	class="category"
+	href={`/categories/${title.toLowerCase()}`}
+	aria-label={`Category ${title} with ${viewersToString(viewers, true)}`}
+>
+	<img src={image} alt={`${title} thumbnail`} />
 	<span>{title}</span>
+	<span class="sr-only">{viewersToString(viewers, true)}</span>
 	<div class="info-container">
 		<span class="title">{title}</span>
 		<div class="viewers">
-			<ViewersGuy />
-			<span>{viewersToString(viewers)} viewers</span>
+			<Fa icon={faUser} size="1.2x" />
+			<span>{viewersToString(viewers, true)}</span>
 		</div>
 	</div>
 </a>
 
 <style lang="scss">
-	@import "../assets/styles/variables.scss";
+	@import "../../assets/styles/variables.scss";
```

Comment:
Can we use $ path syntax?

Reply:
I can't find any documentation for that syntax. I know it works in svelte script tags but I don't think it works in scss style tags.